contracts/DAOTreasury.cdc

```diff
@@ -62,6 +64,12 @@ pub contract DAOTreasury {
       emit ExecuteAction(actionUUID: actionUUID)
     }
 
+    pub fun destroyAction(actionUUID: UInt64) {
```

Comment:
`DestroyAction` should be its own MultiSignAction in `TreasuryActions` and require signers to approve it before executing like other multi-sign actions.  Exposing via `TreasuryPublic` interface would allow anyone to delete a proposed action :)

Reply:
On it!